make_wheels.py

```diff
@@ -266,6 +266,7 @@ def dummy(): """Dummy function for an entrypoint. Zig is executed as a side effe
             ('Classifier', 'Topic :: Software Development :: Compilers'),
             ('Classifier', 'Topic :: Software Development :: Code Generators'),
             ('Classifier', 'Topic :: Software Development :: Build Tools'),
+            ('Classifier', 'Programming Language :: Zig'),
             ('Classifier', 'Programming Language :: Other'),
             ('Classifier', 'Programming Language :: Other Scripting Engines'),
```

Comment:
Hi @whitequark, should we remove the "Other" classifiers now, or would you prefer keeping them?